### ISSUE TITLE
Empty MBus usage unit results in NullPointerException

### ISSUE THREAD

**Issue by WoutProvost** [nielsbasjes/dsmr-tools#5]
Output of a **Landis+Gyr E350 (DSMR 4.2)**:
```
/XMX5LGBBFG1009089532

1-3:0.2.8(42)
0-0:1.0.0(000101010000W)
0-0:96.1.1(4530303330303033313131393539373135)
1-0:1.8.1(000024.487*kWh)
1-0:1.8.2(000030.536*kWh)
1-0:2.8.1(000000.528*kWh)
1-0:2.8.2(000000.000*kWh)
0-0:96.14.0(0002)
1-0:1.7.0(00.000*kW)
1-0:2.7.0(00.000*kW)
0-0:96.7.21(00116)
0-0:96.7.9(00000)
1-0:99.97.0(0)(0-0:96.7.19)
1-0:32.32.0(00011)
1-0:52.32.0(00028)
1-0:72.32.0(00028)
1-0:32.36.0(00000)
1-0:52.36.0(00000)
1-0:72.36.0(00000)
0-0:96.13.1()
0-0:96.13.0()
1-0:31.7.0(000*A)
1-0:51.7.0(000*A)
1-0:71.7.0(000*A)
1-0:21.7.0(00.004*kW)
1-0:41.7.0(00.000*kW)
1-0:61.7.0(00.000*kW)
1-0:22.7.0(00.000*kW)
1-0:42.7.0(00.000*kW)
1-0:62.7.0(00.000*kW)
0-1:24.1.0(003)
0-1:96.1.0(4730303032333430313537323637333134)
0-1:24.2.1(000101010000W)(0000000000)
!6D3C
```
Notice the `0-1:24.2.1` without a unit in the second parentheses.

I've tried:
1. making the `*` and `unit` optional in the ANTLR file:
`'(' value=(FLOAT|INT) ('*' unit=('m3'|'GJ'|'kWh'))? ')'` instead of
`'(' value=(FLOAT|INT) '*' unit=('m3'|'GJ'|'kWh') ')'`.
2. adding a null check in the `visitMBus1Usage` method:
`ctx.unit == null ? "" : ctx.unit.getText()`

Both are needed, but there seems to be another problem somewhere else, as the telegram is still invalid.

**Comment by WoutProvost** [comment]
Found 2 parts where it fails:

1. the parser doesn't know about `0-0:96.13.1`, which is still present in DSMR 4.2, but not in DSMR 5.0.
2. the `fillMBusDataToAttributes` sets `hasSyntaxError` and by extension `isValid` to false, because the unit is an empty string.

**Comment by nielsbasjes** [comment]
I released 0.5 with your improvements.
Thanks!

**Comment by nielsbasjes** [comment]
I've added an extra test to ensure this example keeps working.
